### PR TITLE
Completed Issue-2-view-entries feature

### PR DESCRIPTION
**Pull request by SnowWhiteDad**
Issue-2-view-entries : Ola Sh2

view-entries-1

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    entryId: data.nextEntryId,
+    title: $entryForm.elements.entryTitle.value,
+    photoUrl: $entryForm.elements.photoUrl.value,
+    notesText: $entryForm.elements.entryNotes.value
+  };
+  data.entries.unshift(newData);
+  if (data.nextEntryId === 1) {
+    document.getElementById('entry-list').innerHTML = '';
+  }
+  document.getElementById('entry-list').appendChild(renderEntry(newData));
```
Comment (Zhiwutian) [inline js/main.js:43]: you want to use the prepend method here to add the entry to the start of the ul

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking great Ola, a couple small updates!

File: js/main.js (hunk 1 above)
Comment (SnowWhiteDad) [reply js/main.js]: Done.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**